x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ssl/SSLServiceTests.java

```diff
@@ -873,6 +878,48 @@ public void testWrapTrustManagerWhenInFipsAndExplicitlyConfigured() {
         assertThat(sslService.wrapWithDiagnostics(wrappedTrustManager, sslConfiguration), sameInstance(wrappedTrustManager));
     }
 
+    public void testDeprecationOfTruststoreWithNoTrustedEntries() throws IllegalAccessException {
+        Path pathToEmptyKeystore = getDataPath("/org/elasticsearch/xpack/security/transport/ssl/certs/simple/empty.p12");
```

Comment:
Nit: I wonder whether it would be a more realistic test if this keystore file contains just private keys instead of being truly empty?